src/components/DeliveryDetails/RecipientForm/helpers.ts

```diff
@@ -1,7 +1,16 @@
-const getDate = (date: number) =>
-  new Date(date * 1000).toISOString().split('T')[0];
+const getLocalDate = (timestamp: number) => {
+  const d = new Date(timestamp * 1000);
+  const year = d.getFullYear();
+  const month = String(d.getMonth() + 1).padStart(2, '0');
+  const day = String(d.getDate()).padStart(2, '0');
+  return `${year}-${month}-${day}`;
```

Comment:
I was wondering about that local format because at the end its not a local format (for example in the USA there is mm-dd-yyyy).
I was thinking about `toLocaleDateString()` and `toLocaleTimeString()` methods but if we are based on that yyyy-mm-day format everywhere maybe it will be just problematic